Server/ConcordiaCurriculumManagerTest/UnitTests/Services/CourseServiceTest.cs

```diff
@@ -1,4 +1,5 @@
-﻿using ConcordiaCurriculumManager.DTO.Dossiers;
+﻿using ConcordiaCurriculumManager.DTO.Courses;

```

Comment:
Good tests, but can you also write the controller tests?

Reply:
Will do